api/app/views/spree/api/addresses/_address.json.jbuilder

```diff
@@ -0,0 +1,17 @@
+json.cache! address do
```

Comment:
I know we just want to reflect views as they are, I just want to take note that this is the only place where `cache!` is used without `I18n.locale` key. Probably stuff for another PR.